src/entities/Proposal/types.ts

```diff
@@ -126,6 +135,20 @@ export function getPoiTypeAction(poiType: PoiType) {
   return poiType.split('_')[0] // "add" | "remove"
 }
 
+export function toHiringType(value: string | null | undefined): HiringType | null {
```

Comment:
What about changing this to `isValidHiringType` and return a boolean accordingly??

Reply:
I'd also use the same logic in both Hiring and POI submit pages.

Reply:
I agree that returning boolean is better than returning null. 
But if you need to cast some value to a valid type, it's useful to have a fn that can do it.

A good approach IMHO is something like we have with proposal status:

```typescript
export function toHiringType(value: string | null | undefined, orElse: () => any): HiringType | any {
  return isValidHiringType(value) ? (value as HiringType) : orElse()
}
```

You can always cast something but it forces you to provide an explicit solution when the casting fails (which could be returning a default type, returning null, or throwing an error, reporting, etc).

```typescript 
toHiringType(something, () => null)
toHiringType(something, () => { throw new Error("Invalid hiring type" })
```